src/storage_manifests.py

```diff
@@ -142,3 +143,24 @@ def evaluate(self) -> Optional[str]:
             if not self.config.get(prop):
                 return f"Storage manifests waiting for definition of {prop}"
         return None
+
+class UpdateControllerPlugin(Patch):
+    """Update the controller args in Deployments."""
+
+    def __call__(self, obj):
+        """Update the controller args in Deployments."""
+        if not any(
+            [
+                (obj.kind == "DaemonSet" and obj.metadata.name == "csi-cinder-nodeplugin"),
+                (obj.kind == "Deployment" and obj.metadata.name == "csi-cinder-controllerplugin"),
+            ]
+        ):
+            return
+
+        for container in obj.spec.template.spec.containers:
+            if container.name == "csi-provisioner":
+                for i, val in enumerate(container.args):
+                    if "feature-gates" in val.lower():
+                        container.args[i] = f"feature-gates=Topology={self.manifests.config.get('topology')}"
+                log.info(f"Disabling cinder topology awareness")
```

Comment:
I think that argument needs to be lower-cased

```suggestion
                    if "feature-gates" in val.lower():
                        topology = str(self.manifests.config.get("topology")).lower()
                        container.args[i] = f"feature-gates=Topology={topology}"
                        log.info(f"Configuring cinder topology awareness=%s", topology)
```